File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int>count(26,0);
        if(s.length()!=t.length())return false;
        for(int i=0;i<s.length();i++){
            count[s[i]-'a']++;
            count[t[i]-'a']--;
        }
        for(int i:count){
            if(i!=0)return false;
        }
        return true;
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.java
class Solution {
    public int majorityElement(int[] nums) {
        int res=0;
        int count=0;
        for(int i:nums){
            if(count==0){
                res=i;
            }
            count+=res==i?1:-1;
        }
        return res;
    }
}